frontend/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "check": "npm run lint && npm run prettier",
+    "dev": "next dev -p 3001",
```

Let's not hardcode the port in package.json. Instead, we can just pass in the port as a command line arg from the root package.json for when you start things up with start:all. (see other comment)

```suggestion
    "dev": "next dev",
```